tests/providers/amazon/aws/sensors/test_batch.py

```diff
@@ -100,6 +100,12 @@ def test_execute_failure_in_deferrable_mode(self, deferrable_batch_sensor: Batch
         with pytest.raises(AirflowException):
             deferrable_batch_sensor.execute_complete(context={}, event={"status": "failure"})
 
+    def test_execute_failure_in_deferrable_mode_with_soft_fail(self, deferrable_batch_sensor: BatchSensor):
+        """Tests that an AirflowSkipException is raised in case of error event and soft_fail is set to True"""
+        deferrable_batch_sensor.soft_fail = True
+        with pytest.raises(AirflowException):
```

Comment:
We need to check if AirflowSkipException is raised not its superclass
```suggestion
        with pytest.raises(AirflowSkipException):
```

Reply:
Yep, you're right. Just updated it. Thanks!